### PR TITLE
Add units tab to docs

### LINKED ISSUE #137: Clear documentation of expected parameter input
To make it easier and reduce the chance of mistakes, we should make sure we explicitly spell out the format/functional form and units of the parameters. 

Specific things to note:
distance is in nm
energy is in kJ/mol
angles are radians
harmonic bonds are defined with a factor of 0.5 in front. 
Include the simple conversions from opls-style fourier series to RB.

openmm does have a good page documenting some of this; we should link to it, but also make a summary in the docs folder.

http://docs.openmm.org/7.1.0/userguide/theory.html

### PR DESCRIPTION
**Pull request by jennyfothergill**
### PR Summary:
~With a little more work~ this fixes #137 
- [x] distance is in nm
- [x] energy is in kJ/mol
- [x] angles are radians
- [x] harmonic bonds are defined with a factor of 0.5 in front.
- [x] Include the simple conversions from opls-style fourier series to RB.

related to https://github.com/mosdef-hub/mbuild/pull/858 (The links will not work until both are merged)

### PR Checklist
------------
 - [ ] Includes appropriate unit test(s)
 - [ ] Appropriate docstring(s) are added/updated
 - [ ] Code is (approximately) PEP8 compliant
 - [x] Issue(s) raised/addressed?

### CODE REVIEW

File: readthedocs.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+python:
+  version: 3.7
+  install:
+    - requirements: docs/env-docs.yml
+  system_packages: true
+
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: false
```
Comment (umesh-timalsina) [inline readthedocs.yml:14]: This file is redundant as there's `.readthedocs.yml` in this repo. Is there a specific reason to add this file?

File: docs/units.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+| Quantity |  Units  |
++==========+=========+
+| distance |    nm   |
++----------+---------+
+|   angle  | radians |
++----------+---------+
+|   mass   |   amu   |
++----------+---------+
+|  energy  |  kJ/mol |
++----------+---------+
```
Comment (umesh-timalsina) [inline docs/units.rst:17]: Might be nice to link `OpenMM` [units](http://docs.openmm.org/latest/userguide/theory.html#units) here as well.

**Review by umesh-timalsina — COMMENTED** [review]
General comments. Looks Good.

File: readthedocs.yml (hunk 1 above)
Comment (jennyfothergill) [reply readthedocs.yml]: 🤦‍♀️ I think I missed that! Fixed now

File: docs/units.rst (hunk 2 above)
Comment (jennyfothergill) [reply docs/units.rst:17]: good call!

**Comment by jennyfothergill** [comment]
it seems like without the readthedocs.yml the build is no longer triggered in the PR. @umesh-timalsina can you fix this in the rtd dashboard?

**Comment by umesh-timalsina** [comment]
@jennyfothergill. They should be now.

**Review by justinGilmer — APPROVED** [review]

**Review by daico007 — APPROVED** [review]

**Pull request merged**